help/en/releasenotes/current-draft-note.shtml

```diff
@@ -585,7 +585,7 @@
    <h3>Web Access</h3>
         <a id="WA" name="WA"></a>
         <ul>
-            <li></li>
+            <li>new parm ("protect=yes") to treat web-panel as read-only</li>
```

Comment:
I suggest that you change `parm` to `parameter`. For those who only reads the current-draft-note, it might be difficult to understand what you mean with parm.

Reply:
No worries, changed.